lib/content-services/src/lib/tag/tag-node-list.component.ts

```diff
@@ -87,4 +108,35 @@ export class TagNodeListComponent implements OnChanges, OnDestroy, OnInit {
             this.refreshTag();
         });
     }
+
+    displayAllTags(): void {
+        this.limitTagsDisplayed = false;
+        this.refreshTag();
+    }
+
+    private calculateTagsToDisplay() {
```

Comment:
please add void

Reply:
we need to decide on the return types and enable that in the linter... adding void everywhere was the "strict" mode of the Typescript, not sure if they suggest that these or not

Reply:
So not sure if we should follow that for now or not :)

Reply:
That's good question, I saw that Denys added some comment to that Jira but nobody replied yet

Reply:
for now I suggest we allow "void" based on personal preferences, otherwise it should go to linter rules to be enforced